### DISCUSSION TITLE
Radiation budget at different layers (heights) within the canopy.

### DISCUSSION THREAD

**Discussion by hamiddashti** [eradiate/eradiate#376] [Q&A] [answered]
Hi, I was wondering if the model outputs the radiation budget between two arbitrary planes within the canopy. Let's say I have a tree with a 3 m height and I am interested in the radiation budget in the layer that is between height=1m and height=2m. I want to know how much energy is being absorbed and scattered downward and upward  in this specific layer.

My understanding is based on [this](https://github.com/eradiate/eradiate/discussions/363) discussion the model does not have this feature yet? 

Thanks

**Comment by leroyvn** [comment, marked as answer]
Hi @hamiddashti, unfortunately, this is not supported. I think the workflow would be similar to the one suggested in the discussion you are talking about.
